docs/ad/api.md

```diff
@@ -215,67 +258,67 @@ POST /_opendistro/_anomaly_detection/results/_search
 
 ```json
 {
-  "took" : 9,
-  "timed_out" : false,
-  "_shards" : {
-    "total" : 25,
-    "successful" : 25,
-    "skipped" : 0,
-    "failed" : 0
+  "took": 9,
+  "timed_out": false,
+  "_shards": {
+    "total": 25,
+    "successful": 25,
+    "skipped": 0,
+    "failed": 0
   },
-  "hits" : {
-    "total" : {
-      "value" : 2,
-      "relation" : "eq"
+  "hits": {
+    "total": {
+      "value": 2,
+      "relation": "eq"
     },
-    "max_score" : 1.0,
-    "hits" : [
+    "max_score": 1,
+    "hits": [
       {
-        "_index" : ".opendistro-anomaly-results-history-2019.11.21-1",
-        "_type" : "_doc",
-        "_id" : "GO0UkW4BbunK2JwuvMUQ",
-        "_version" : 1,
-        "_seq_no" : 35,
-        "_primary_term" : 1,
-        "_score" : 1.0,
-        "_source" : {
-          "detector_id" : "dAaHjG4B5Uh30OvgBjjc",
-          "anomaly_score" : 2.4603645209872402,
-          "start_time" : 1574391785478,
-          "confidence" : 0.8429920264756339,
-          "feature_data" : [
+        "_index": ".opendistro-anomaly-results-history-2019.11.21-1",
+        "_type": "_doc",
+        "_id": "GO0UkW4BbunK2JwuvMUQ",
+        "_version": 1,
+        "_seq_no": 35,
+        "_primary_term": 1,
+        "_score": 1,
+        "_source": {
+          "detector_id": "dAaHjG4B5Uh30OvgBjjc",
+          "anomaly_score": 2.4603645209872402,
+          "start_time": 1574391785478,
+          "confidence": 0.8429920264756339,
+          "feature_data": [
             {
-              "feature_id" : "value_sum",
-              "feature_name" : "Value Sum",
-              "data" : 70.1979163525
+              "feature_id": "value_sum",
+              "feature_name": "Value Sum",
+              "data": 70.1979163525
             }
           ],
-          "end_time" : 1574392085478,
-          "anomaly_grade" : 0.6303142329020289
+          "end_time": 1574392085478,
```

Comment:
`start_time` -> `data_start_time`
`end_time` -> `data_end_time`
And , add two more time fields: `execution_start_time` and `execution_end_time`

Reply:
Added.